backup.sh

```diff
@@ -35,8 +35,11 @@ set -xeuo pipefail
 if [ "${OCP_BACKUP_S3}" = "true" ]; then
     # prepare & push backup to S3
 
-    # update CA trust
-    update-ca-trust
+    # add custom CA if any and update CA trust
+    if [ "${OCP_BACKUP_S3_CA}" ]; then
+        echo -n "${OCP_BACKUP_S3_CA}" > /etc/pki/ca-trust/source/anchors/ca.crt
+        update-ca-trust
+    fi
```

Comment:
why can't we just mount the ca to `/etc/pki/ca-trust/source/anchors/ca.crt` from a ConfigMap?

Reply:
this will imply having a dedicated configMap for just the CA, I wanted to avoid that, what do you think?

Reply:
afair that's what the previous `update-ca-trust` call is for

Reply:
what about this @hairmare?

Reply:
afaict we can create configure the cluster to know about the CA and create an empty configmap with the `config.openshift.io/inject-trusted-cabundle="true"` annotation using this chaart and then magick should happen:

https://github.com/openshift/openshift-docs/blob/main/modules/certificate-injection-using-operators.adoc

Reply:
Specifically, the cluster *should* already know about the right CA. We use the same trust root that for basics like accessing internal registries. it is usually configured during installation, moreso in air-gapped cases than others.

The certificate injection feature seemingly aims to cover the same need that [trust-manager](https://cert-manager.io/docs/trust/trust-manager/) wants to cover, but as a CNCF (sub-)project. The [openshift/service-ca-operator](https://github.com/openshift/service-ca-operator) seems to be RedHat's upstream for their implementation that most likely predates trust-manager.